examples/quick_template_score_calc.ipynb

```diff
@@ -528,7 +520,7 @@
    "source": [
     "temperature_score = TemperatureScore(\n",
     "    time_frames = [ETimeFrames.LONG],\n",
-    "    scopes=[EScope.S1S2],\n",
```

Comment:
From slack chat with Michael:

I changed CBRE S3 emissions in the input data so it’s just slowly growing up from 20k to 21k to 22k, instead of the monstrous increase that was a result of new reporting standards.  I would expect this would change the S3 score and not the S1S2 score, but it has the opposite effect (and no change to the S3 score):
~~~~
 150    CBRE    LONG    S1S2    1.22
 151    CBRE    LONG    S3    1.5
~~~~

Reply:
This finding is caused by the other finding in Jupiter notebook for S3, so I will resolve this duplicate entry